src/eatery_db/collegetown_eatery.py

```diff
@@ -89,5 +89,9 @@ def parse_collegetown_hours(data_json, eatery_model):
                             start_time=start,
                         )
                     )
+                if not new_events:
+                    new_operating_hours.append(
+                        CollegetownEateryHour(eatery_id=eatery_model.id, date=new_date.isoformat(),)
```

Comment:
What does the extra comma at the end of the line do?

Reply:
Nothing, it got removed with Black.